OpenSim/Common/XMLDocument.cpp

```diff
@@ -54,7 +54,8 @@ using namespace std;
 // 30502 for Changing serialization of Geometry
 // 30503 for Changing serialization of Ground
 // 30505 for Changing serialization of Joint to create offset frames
-const int XMLDocument::LatestVersion = 30505;   
+// 30506 for Changing serialization of PrescribedForce to use Connectors
+const int XMLDocument::LatestVersion = 30506;   
```

Comment:
Is it necessary to increment the version number for every change vs. waiting until we are done for 4.0 and increment to a new version number then?